cmd/neofs-node/object.go

```diff
@@ -256,14 +256,13 @@ func initObjectService(c *cfg) {
 		searchsvcV2.WithKeyStorage(keyStorage),
 	)
 
-	sPut := putsvc.NewService(&transport{clients: putConstructor},
+	sPut := putsvc.NewService(&transport{clients: putConstructor}, c,
```

Comment:
btw, why not an option? looks strange when there are position args and options but _both_ are required in fact

Reply:
to not forget to pass it. I agree that in the end this should be brought to a general form